services/transaction-statistics/jobs/transactionStatistics.js

```diff
@@ -28,7 +28,8 @@ module.exports = [
 	{
 		name: 'refresh.transactions.statistics',
 		description: 'Keep the transaction statistics up-to-date',
-		schedule: '*/30 * * * *', // Every 30 min
+		interval: config.jobs.refreshTransactionStats.interval,
+		schedule: config.jobs.refreshTransactionStats.schedule,
```

Comment:
The idea is to add support for both interval and schedule to the end users. And in our current implementation, default value for interval is 0 which means the job will schedule instead of interval. Also this allow users to configure as per their requirements. We can discuss offline if you need any clarifications.